src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+# SPRING
+spring.application.name=teslaris
+server.port=8081
+server.address=localhost
+
+# DATA SOURCE
+spring.datasource.url=jdbc:postgresql://localhost:5432/TeslaRISDB
+spring.datasource.username=${DB_USERNAME:postgres}
+spring.datasource.password=${DB_PASSWORD:root}
+
+# HIBERNATE
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
should be careful later with this create mode, let's keep that for now, and change later